jerry-main/main-unix.c

```diff
@@ -121,6 +121,26 @@ print_help (char *name)
                       name);
 } /* print_help */
 
+static void
+print_unhandled_exception (jerry_value_t error_value)
+{
+  assert (jerry_value_has_error_flag (error_value));
+
+  jerry_value_clear_error_flag (&error_value);
+  jerry_value_t err_str_val = jerry_value_to_string (error_value);
+
+  jerry_char_t err_str_buf[256];
+  jerry_size_t err_str_size = jerry_get_string_size (err_str_val);
+
+  assert (err_str_size < 256);
```

Comment:
Either don't print the error when the size is > 256 or dynamically allocate a buffer.